package.json

```diff
@@ -75,8 +77,17 @@
   "release": {
     "branch": "master",
     "debug": false,
-    "verifyConditions": {
-      "path": "./node_modules/@krux/condition-jenkins"
-    }
+    "prepare": [
```

Comment:
is `verifyConditions` no more needed?

Reply:
It used to check the existence of env variables to find whether we running from CI env or not. Now it is not required.